internal/pkg/cli/app_delete.go

```diff
@@ -221,6 +231,15 @@ func (o *deleteAppOpts) Execute() error {
 	return nil
 }
 
+func (o *deleteAppOpts) askAppName() error {
```

Comment:
Not an action item for you at all, but I just want to share an alternative way of writing this, which is my personal preference for the time being. 


```suggestion
func (o *deleteAppOpts) Ask() error {
   if err := o.validateOrAskAppName(); err != nil { return err }
   if o.skipConfirmation { return nil }
   // ... the rest
}
func (o *deleteAppOpts) validateOrAskAppName() error {
      if o.name != "" {
           _, err := o.store.GetApplication(o.name)
	   return err
      }
      name, err := o.sel.Application(appDeleteNamePrompt, "")
	if err != nil {
		return fmt.Errorf("select application name: %w", err)
	}
	o.name = name
	return nil
}
```

This way, it is easy to visualize the _high-level steps_ taken in `Ask` - that we first do something with the app name, then we do something asking users to confirm the deletion.